src/crypto/index.js

```diff
@@ -246,6 +246,18 @@ Crypto.prototype.uploadDeviceKeys = function() {
     });
 };
 
+/**
+ * Stores the current key count which will be handled postponed (in a call of
+ * _onSyncCompleted)
+ *
+ * @param {Number} currentKeyCount to be stored and handled later
+ */
+Crypto.prototype.updateCurrentKeyCount = function(currentKeyCount) {
+    if (isFinite(currentKeyCount)) {
```

Comment:
I'd say it's up to the caller of the API to check that the parameter is valid. If you want to double-check it for robustness, throw an exception:

```
if (!isFinite(currentKeyCount)) {
    throw new Error("Invalid parameter to updateCurrentKeyCount");
}
```

or similar